omnigibson/utils/usd_utils.py

```diff
@@ -602,10 +604,10 @@ def clear(cls):
         """
         # Remove all the collision group prims
         for col_group_prim in cls.ACTIVE_COLLISION_GROUPS.values():
-            og.sim.stage.RemovePrim(col_group_prim.GetPath().pathString)
+            delete_or_deactivate_prim(col_group_prim.GetPath().pathString)
 
         # Remove the collision groups tree
-        og.sim.stage.RemovePrim("/World/collision_groups")
+        delete_or_deactivate_prim("/World/collision_groups")
```

Comment:
These are problematic: in a multiple envs context delete_or_deactivate_prim will get called lots of times which will mean it will call its downstream update_handles implication multiple times. Instead, let's make it accept a list of prim paths (delete_or_deactivate_prims), call it with a flattened list of prim paths everywhere it's called in an obvious loop, and make it call update handles just once.